### PR TITLE
bees now die when stinging a living target

### PR DESCRIPTION
**Pull request by Ohagi-Chan**
## About The Pull Request
Bees now die after stinging a living target, to fall in line more with reality, and to reduce the lethality of what should not be quite so lethal a throwaway mob.

## Why It's Good For The Game
Ided
Also, this is an improvement to "positive" bees, who would otherwise just OD their targets, which completely defeats the point of making bees to be helpful.

## Changelog
:cl:
balance: bees die after stinging a living target
/:cl:

### CODE REVIEW

**Comment by Dawson1917** [comment]
you should buff them considering theyre intended to be lethal in many cases and there are antag items that use them

**Comment by Ohagi-Chan** [comment]
this is also a buff to omnizine/bicaridine bees etc, which won't be accidentally lethal anymore.
as they should have been all along.

File: austation/code/modules/mob/living/simple_animal/hostile/bees.dm [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/mob/living/simple_animal/hostile/poison/bees/AttackingTarget()
+	..()
+	if isliving(target)
```
Comment (TheFakeElon) [inline austation/code/modules/mob/living/simple_animal/hostile/bees.dm:3]: go to jail do not pass go

**Comment by TheFakeElon** [comment]
How in the flying fuck does that compile

File: austation/code/modules/mob/living/simple_animal/hostile/bees.dm (hunk 1 above)
Comment (Ohagi-Chan) [reply austation/code/modules/mob/living/simple_animal/hostile/bees.dm:3]: what's the problem?

**Review by TheFakeElon — APPROVED** [review]

**Pull request merged**